cornflow-server/cornflow/endpoints/execution.py

```diff
@@ -184,7 +186,9 @@ def post(self, **kwargs):
 
         # Validate config before running the dag
         config_schema = DeployedDAG.get_one_schema(config, schema, CONFIG_SCHEMA)
-        config_errors = json_schema_validate_as_string(config_schema, kwargs["config"])
+        new_config, config_errors = json_schema_extend_and_validate_as_string(
+            config_schema, kwargs["config"]
+        )
```

Comment:
@marioncottard This is the new validation that adds the default values (if needed) before validating the schema